make.py

```diff
@@ -176,7 +176,7 @@ def build_track(output_directory, track, trackno):
         argv += ["-metadata", f"ARTIST={track['artist']}"]
 
     if lyrics_file:
-        with open(os.path.join(os.path.dirname(__file__), lyrics_file)) as fd:
+        with open(os.path.join(os.path.dirname(__file__), lyrics_file), encoding="utf8") as fd:
```

Comment:
I generally don't write text past the 79th column, so please write this as follows  to match the style of the rest of the code:

```
        with open(os.path.join(os.path.dirname(__file__), lyrics_file),
          encoding="utf-8") as fd:
```